### ISSUE TITLE
ailed opening required '/var/www/grav/grav/vendor/symfony/polyfill-iconv/bootstrap.php'

### ISSUE THREAD

**Issue by Corwin7** [getgrav/grav#606]
After upgrading grav to 1.0.8 (`* master  b48449d Merge branch 'release/1.0.8'`) I am now getting 500 error with this in the Nginx error log:

```
2016/01/12 01:18:55 [error] 29578#29578: *11155 FastCGI sent in stderr: "PHP message: PHP Warning:  require(/var/www/grav/grav/vendor/symfony/polyfill-iconv/bootstrap.php): failed to open stream: No such file or directory in /var/www/grav/grav/vendor/composer/autoload_real.php on line 54
PHP message: PHP Fatal error:  require(): Failed opening required '/var/www/grav/grav/vendor/symfony/polyfill-iconv/bootstrap.php' (include_path='.:/usr/share/php:/usr/share/pear') in /var/www/grav/grav/vendor/composer/autoload_real.php on line 54" while reading response header from upstream, client: 157.55.39.37, server: www.sendthemtomir.com, request: "GET / HTTP/1.1", upstream: "fastcgi://127.0.0.1:9001", host: "www.sendthemtomir.com"
```

```
╰─➤  bin/composer.phar update 
Loading composer repositories with package information
Updating dependencies (including require-dev)         
  - Updating erusev/parsedown (dev-master e7443a2 => 1.6.0)

                                                                                                                                               
  [RuntimeException]                                                                                                                           
  The .git directory is missing from /var/www/grav/grav/vendor/erusev/parsedown, see https://getcomposer.org/commit-deps for more information 
```

**Comment by Corwin7** [comment]
I think these are two separate issues. I did a `git checkout  8ca14c7` but still didn't work. I restored last night's backups and it worked, even though git was still at 8ca14c7. `bin/composer.phar` has been modified from git in both.

The site is back to working with 1.0.7, but I still get an error from `bin/composer.phar update`.

Grav is amazing. :)

**Comment by rhukster** [comment]
How did you upgrade to 1.0.8?  

If it was via Github or Composer, Try this:

```
$ rm -Rf vendor/
$ bin/composer.phar install --no-dev -o
```

**Comment by rhukster** [comment]
BTW, the Grav update packages (udpated via GPM) includes a whole new vendor folder so this is not a problem via that route.

**Comment by Corwin7** [comment]
You're dead on @rhukster. :) I upgraded via git, and removing the vendor folder and the `bin/composer.phar install --no-dev -o` fixed everything with 1.0.8.

Thank you once again for the amazing and lightning fast support.
